client/jetpack-connect/plans-placeholder.js

```diff
@@ -10,19 +10,18 @@ import { times, random } from 'lodash';
  * Internal dependencies
  */
 import Main from 'components/main';
+import FormattedHeader from 'components/formatted-header';
 
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 const placeholderContent = (
 	<Main className="jetpack-connect__hide-plan-icons" wideLayout>
 		<div className="jetpack-connect__plans placeholder">
-			<header className="formatted-header">
-				<h1 className="formatted-header__title">
-					<span className="placeholder-text">Your site is now connected!</span>
-				</h1>
-				<p className="formatted-header__subtitle">
+			<FormattedHeader
+				headerText={ <span className="placeholder-text">Your site is now connected!</span> }
+				subHeaderText={
```

Comment:
I am not quite sure why this `subHeaderText` is broken down into three lines when I commit. Is that something handled by `prettier` rules?

Reply:
Yep, the line is too long and Prettier breaks it into multiple shorter ones.

Reply:
Okay, good to know that is okay. 👍🏽